### ISSUE TITLE
20NG Training Set Reported Results Question

### ISSUE THREAD

**Issue by jharrang** [allenhaozhu/SSGC#11]
For your 20NG results reported in [your original paper](https://openreview.net/pdf?id=CYO5T-YjWZV) (`88.6± 0.1 `), was the model trained on the full public 20NG `train` set, of size 11314, or were the reported results generated using the training code currently in this repo [here](https://github.com/allenhaozhu/SSGC/blob/fb833a7149f439a15fad45c9ebaca30aac46be3f/DocumentClassification/train_cpu.py#L69) and [here](https://github.com/allenhaozhu/SSGC/blob/fb833a7149f439a15fad45c9ebaca30aac46be3f/DocumentClassification/train.py#L69), which appears to exclude the validation set from the data used to train the model? (The latter would use a training set of size 10183).

Thanks!

**Comment by allenhaozhu** [comment]
Yes, we exclude the validation set. This code is released a few days ago because some guys sent me a mail asking the detail of text classification. And after I simply checked the results, I release codes for text classification. If there is any issue please let me know.

**Comment by vencentDebug** [comment]
RuntimeError: Expected all tensors to be on the same device, but found at least two devices, cpu and cuda:0! (when checking argument for argument target in method wrapper_nll_loss_forward)

 use parser.add_argument('--no-cuda', action='store_true', default=True,
                    help='Disables CUDA training.')
args.device = 'cpu'

**Comment by vencentDebug** [comment]
when I use only "CPU", it notices the problem. Then I use "GPU", it also happens another problem.     train_feats = torch.spmm(adj, train_feats)
RuntimeError: Expected all tensors to be on the same device, but found at least two devices, cuda:0 and CPU! (when checking argument for argument mat2 in method wrapper__mm)
